File: src/Workflow.Web.Service/Workflow/app/model/ActivityConfig.js
Ext.define('Workflow.model.ActivityConfig', {
	extend: 'Workflow.model.Base',
	
	fields: [
		{ name: 'id', type: 'int' },
        { name: 'requestCode', type: 'string' },
        { name: 'activity' },
		{ name: 'actions'}
	]
});

File: src/Workflow.Domain/Entities/ActivityConfig.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Text;
using System.Threading.Tasks;

namespace Workflow.Domain.Entities
{
    [DataContract]
    public class ActivityConfig
    {
        [DataMember(Name = "id")]
        public int Id { get; set; }

        [DataMember(Name = "requestCode")]
        public string RequestCode { get; set; }

        [DataMember(Name = "activity")]
        public string Activity { get; set; }

        [DataMember(Name = "actions")]
        public string[] Actions { get; set; }
    }
}
